filter_test.go

```diff
@@ -584,7 +582,14 @@ func TestFilterClsActBpfAddDel(t *testing.T) {
 	if q, ok := qdiscs[0].(*GenericQdisc); !ok || q.Type() != "clsact" {
 		t.Fatal("qdisc is the wrong type")
 	}
+	return qdiscs[0], link
+}
+
+func TestFilterClsActBpfAddDel(t *testing.T) {
+	tearDown := setUpNetlinkTest(t)
```

Comment:
I no longer have a 3.10 kernel VM at hand to check this, but I am assuming this test would fail on older kernel.

Can you please add the call
```
minKernelRequired(t, x, y)
```
at the beginning of this test function, and move the same call at line 572 to the top of the new test function.

Thanks